### PR TITLE
Add comments to hook class

### PR DESCRIPTION
**Pull request by jf-viguier**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add comments to Hook class
| Type?             | refacto
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A
| Possible impacts? | None, just comments
| Sponsor company   | Creabilis.com

### CODE REVIEW

File: classes/Hook.php [hunk 1]
```diff
@@ -154,6 +154,13 @@ public static function normalizeHookName($hookName)
         return $hookNamesByAlias[$loweredName] ?? $hookName;
     }
 
+    /**
+     * Return true If the hook name starts with "display"
+     *
+     * @param string hook_name The name of the hook to check
```
Comment (kpodemski) [inline classes/Hook.php]:
```suggestion
     * @param string $hook_name The name of the hook to check
```

File: classes/Hook.php [hunk 2]
```diff
@@ -154,6 +154,13 @@ public static function normalizeHookName($hookName)
         return $hookNamesByAlias[$loweredName] ?? $hookName;
     }
 
+    /**
+     * Return true If the hook name starts with "display"
```
Comment (kpodemski) [inline classes/Hook.php]:
```suggestion
     * Return true if the hook name starts with "display"
```

File: classes/Hook.php [hunk 3]
```diff
@@ -154,6 +154,13 @@ public static function normalizeHookName($hookName)
...
+     *
+     * @param string hook_name The name of the hook to check
+     *
+     * @return void
```
Comment (kpodemski) [inline classes/Hook.php]:
```suggestion
     * @return bool
```

File: classes/Hook.php [hunk 4]
```diff
@@ -221,6 +228,8 @@ public static function getIdByName($hookName, bool $withAliases = true, bool $re
     /**
      * Return hook ID from name.
      *
+     * @return string|false Hook name, or false if it doesn't exist
```
Comment (kpodemski) [inline classes/Hook.php]: It returns an exception if the hook does not exist, it never returns false

**Review by kpodemski — CHANGES_REQUESTED** [review]

File: classes/Hook.php (hunk 4 above)
Comment (jf-viguier) [reply classes/Hook.php]: ok thanks

**Review by FabienPapet — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @jf-viguier

**Pull request merged**